common/thorhelper/thorstrand.cpp

```diff
@@ -277,8 +277,8 @@ RowBlockAllocator::RowBlockAllocator(roxiemem::IRowManager & rowManager, size32_
 
     size_t classSize = sizeof(RoxieRowBlock) - RoxieRowBlock::numDummyDynamicRows * sizeof(void *);
     size_t requestedSize = classSize + minRowsPerBlock * sizeof(void*);
-    roxiemem::RoxieHeapFlags heapFlags = roxiemem::RHFunique|roxiemem::RHFnofragment;
-    heap.setown(rowManager.createFixedRowHeap(requestedSize, 0, heapFlags, 0));
```

Comment:
This use of 0 meant that the allocations would return null if there was not enough memory (and would not cause other things to spill) - that is not what was wanted.